xbmc/utils/EGLUtils.h

```diff
@@ -144,3 +147,43 @@ class CEGLContextUtils
   EGLContext m_eglContext;
   EGLConfig m_eglConfig;
 };
+
+/* --- CEGLImage ---------------------------------------------*/
+
+class CEGLImage
+{
+public:
+  static const int MAX_NUM_PLANES{3};
```

Comment:
`AVDRMFrameDescriptor` can carry up to 4 planes and EGL should also support up to 4 planes using `EGL_EXT_image_dma_buf_import_modifiers` at https://github.com/KhronosGroup/EGL-Registry/blob/master/api/EGL/eglext.h#L755-L774, something for future?

Reply:
Something for the future I think. Amlogic mali doesn't seem to support modifiers at this time.